src/scripts/settings.js

```diff
@@ -389,10 +398,23 @@ const Settings = {
               checked = 'checked';
 
               if (!Settings.origins[key]) {
+                tmpkey = key;
+
+                // Special case for toggl-button-demo
+                if (key === 'dom-integration') {
+                  tmpkey = TogglOrigins[key].url
+                    .replace('*://*.', '')
+                    .replace('*://', '')
+                    .replace('/*', '');
+                }
+
                 // Handle cases where subdomain is used (like web.any.do, we remove web from the beginning)
-                tmpkey = key.split('.');
-                tmpkey.shift();
-                tmpkey = tmpkey.join('.');
+                if (TogglOrigins[key].url.match(/\/\/\*\./)) {
```

Comment:
If you think you could name the regex, it'd be much appreciated.
It's hard to read out immediatelly what the `/\/\/\*\./` is 😄, beside looking like a Sioux village 😄